src/test/java/eu/europeana/metis/sandbox/service/problempatterns/PatternAnalysisServiceImplIT.java

```diff
@@ -1,12 +1,7 @@
 package eu.europeana.metis.sandbox.service.problempatterns;
 
 import static org.apache.commons.lang3.StringUtils.isNotBlank;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertLinesMatch;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
-import static org.junit.jupiter.api.Assertions.assertThrows;
-import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Perhaps IntelliJ did it automatically but please keep the previous imports instead of doing it like this

Reply:
oeps, yes. somehow  intelliJ was desconfigured. I reconfigured the coding style again.